simulation/traffic_simulator/include/traffic_simulator/api/api.hpp

```diff
@@ -97,21 +95,13 @@ class API
   auto spawn(
     const std::string & name, const Pose & pose,
     const traffic_simulator_msgs::msg::VehicleParameters & parameters,
-    const std::string & behavior = VehicleBehavior::defaultBehavior())
+    const std::string & behavior = VehicleBehavior::defaultBehavior(), std::string model3d = "")
```

Comment:
```suggestion
    const std::string & behavior = VehicleBehavior::defaultBehavior(), const std::string & model3d = "")
```

Reply:
According to  [ZMQ ss2 <-> AWSIM/simple_simulator document](https://docs.google.com/document/d/1lQIVPp-eHlSMdf8ocGOOIWrZQRSbBYOB6BIkStUJEbo/edit) the message format for `SpawnVehicleEntityRequest` assumes that `VehicleParameters` and `asset_key` are separated. In order to keep the consistency the `model3d` (the same meaning as `asset_key`) hasn't been added to the [VehicleParameters](https://github.com/tier4/scenario_simulator_v2/blob/master/simulation/traffic_simulator_msgs/msg/VehicleParameters.msg) class.

If requested, I can add the `model3d` to the `VehicleParameters`.

Reply:
OK.
I understand, thanks.